Lib/KeychainAccess/Keychain.swift

```diff
@@ -1669,7 +1669,7 @@ extension CFError {
         let code = CFErrorGetCode(self)
         let userInfo = CFErrorCopyUserInfo(self) as! [String: Any]
 
-        return NSError(domain: domain, code: code, userInfo: userInfo)
```

Comment:
CI throws a warning:

> conditional cast from '[String : Any]' to '[String : Any]' always succeeds

…so apparently this is redundant. I don't see a value conversion error in Xcode 9 either, and I'd be very surprised to see it, given line 1670.

I'd vote to close this PR.